src/fides/connectors/aws.py

```diff
@@ -244,6 +244,7 @@ def create_redshift_systems(
             fides_key=cluster["ClusterIdentifier"],
             name=cluster["ClusterIdentifier"],
             description=f"Fides Generated Description for Redshift Cluster: {cluster['ClusterIdentifier']}",
+            meta={(pair["Key"], pair["Value"]) for pair in cluster["Tags"]},
```

Comment:
It looks like we may need to account and handle the scenario where `Tags` doesn't exist (You can see the [checks here](https://github.com/ethyca/fides/actions/runs/9914563221/job/27394125529?pr=4998#step:7:872) )